spring-data-cassandra/src/main/java/org/springframework/data/cassandra/core/ExecutableSelectOperation.java

```diff
@@ -102,6 +102,26 @@
 		 */
 		SelectWithProjection<T> inTable(CqlIdentifier table);
 
+		/**
```

Comment:
This interface is implemented by Spring Data Cassandra only, so we don't need to add default methods for backward compatibility.